common/file/archive/zip_test.go

```diff
@@ -33,60 +36,39 @@ func TestUnZip(t *testing.T) {
 
 func TestZip(t *testing.T) {
 	tempDir := t.TempDir()
-	singleFile := filepath.Join("..", "..", "..", "testdata", "configtest.json")
 	outFile := filepath.Join(tempDir, "out.zip")
-	err := Zip(singleFile, outFile)
-	if err != nil {
-		t.Fatal(err)
-	}
+	err := Zip(filepath.Join("..", "..", "..", "testdata", "configtest.json"), outFile)
+	require.NoError(t, err, "Zip should not error")
 	o, err := UnZip(outFile, tempDir)
-	if err != nil {
-		t.Fatal(err)
-	}
-	if len(o) != 1 {
-		t.Fatalf("expected 1 files to be extracted received: %v ", len(o))
-	}
+	require.NoError(t, err, "UnZip should not error")
+	assert.Len(t, o, 1, "Should extract 1 file")
 
-	folder := filepath.Join("..", "..", "..", "testdata", "http_mock")
+	folder := filepath.Join("..", "..", "..", "testdata", "gctscript")
 	outFolderZip := filepath.Join(tempDir, "out_folder.zip")
 	err = Zip(folder, outFolderZip)
-	if err != nil {
-		t.Fatal(err)
-	}
+	require.NoError(t, err, "Zip should not error")
 	o, err = UnZip(outFolderZip, tempDir)
-	if err != nil {
-		t.Fatal(err)
-	}
+	require.NoError(t, err, "UnZip should not error")
 	var found bool
 	for i := range o {
-		if filepath.Base(o[i]) == "binance.json" {
+		if filepath.Base(o[i]) == "timer.gct" {
 			found = true
 		}
 	}
-	if !found {
-		t.Fatal("could not find file in zip")
-	}
-
-	if expected := 6; len(o) < expected {
-		t.Fatalf("expected at least %v files to be extracted, received: %v ", expected, len(o))
-	}
+	assert.True(t, found, "Should find a gctscript in the zip")
+	assert.Equal(t, 9, len(o), "Should extract 6 files")
```

Comment:
```suggestion
	assert.Equal(t, 9, len(o), "Should extract 9 files")
```

OR

```suggestion
	assert.GreaterOrEqual(t, len(o), 6, "Should extract at least 6 files")
```

The second gives leeway in case scripts are added in the future which I believe was the original intent. Happy with either.

Reply:
Fixed 76a4cd64